go/logger/standard.go

```diff
@@ -350,3 +349,15 @@ func PickFirstError(errors ...error) error {
 func (log *Standard) SetExternalHandler(handler ExternalHandler) {
 	log.externalHandler = handler
 }
+
+type UnforwardedLogger Standard
+
+func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
+	return (*UnforwardedLogger)(log)
+}
+
+func (log *UnforwardedLogger) Debug(s string, args ...interface{})   { log.internal.Debug(s, args...) }
+func (log *UnforwardedLogger) Error(s string, args ...interface{})   { log.internal.Error(s, args...) }
```

Comment:
Golang question: `Standard.Errorf` calls `Standard.Error`. `UnforwardedLogger` overrides `Error` but not `Errorf`. Does `UnforwardedLogger.Errorf` call `UnforwardedLogger.Error` or `Standard.Error`?